datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GraphQLEngine.java

```diff
@@ -33,18 +34,20 @@
  *
  * <p>In addition, it provides a simplified 'execute' API that accepts a 1) query string and 2) set of variables.
  */
-@Data
+@Getter
```

Comment:
Can we just implement the getter methods manually, instead of using lombok? In the future if we added a new `private final` member variable we'd automatically generate the getter, which may not be what we want (eg. if we want to hide internal impl details).